_pages/contribute.md

```diff
@@ -40,14 +40,19 @@ classes: wide2
    submission_details:
      resources: # List any resources associated with the contribution. Not all sections are required
        papers:
-         - Link to paper
+         - title: Link to paper text
```

Comment:
I'm wondering if this wording could be slightly clearer, I could see myself reading this and thinking that "Link to paper text" refers to the URL, rather than the link text.